readme.md

```diff
@@ -430,6 +430,7 @@
 - [Vehicle Security and Car Hacking](https://github.com/jaredthecoder/awesome-vehicle-security)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security) - Security of web apps & services.
 - [Lockpicking](https://github.com/meitar/awesome-lockpicking) - The art of unlocking a lock by manipulating its components without the key.
+- [IOCS](https://github.com/sroberts/awesome-iocs) - Indicators of Compromise for detecting intrusions.
```

Comment:
I think the title should be `Indicator of compromise (IoC)`. `IoC` is too ambiguous and meaningless just alone. It also needs a better description.